e2e-tests/portal/cypress/e2e/journeys/maker/proceed-to-review/proceed-to-review.spec.js

```diff
@@ -1,34 +1,27 @@
-const pages = require('../../../pages');
+import { DEAL_SUBMISSION_TYPE, FACILITY_STAGE } from '@ukef/dtfs2-common';
+
+const { contract, dashboardDeals } = require('../../../pages');
 const partials = require('../../../partials');
 const fillBondForm = require('../../maker-bond/fill-bond-forms');
 const fillLoanForm = require('../../maker-loan/fill-loan-forms');
 const MOCK_USERS = require('../../../../../../e2e-fixtures');
-const dealWithNoFacilities = require('../submit-issued-facilities-for-review/dealWithNoFacilities');
 
 const { BANK1_MAKER1, ADMIN } = MOCK_USERS;
 
 context('Ensure proceed to review button is only visible once facilities are in eligible for submission', () => {
-  let deal;
-
-  before(() => {
-    cy.insertOneDeal(dealWithNoFacilities, BANK1_MAKER1).then((insertedDeal) => {
-      deal = insertedDeal;
-    });
+  beforeEach(() => {
+    cy.createBssEwcsDeal({});
   });
```

Comment:
does this need to be `beforeEach`?

Reply:
We need to mention for each it block if taken out of beforeEach